### ISSUE TITLE
Get Javascript error during rendering

### ISSUE THREAD

**Issue by BirtheG** [RonnyWeiss/Apex-D3-Organization-Chart#4]
Hallo Ronny,
If I try the plugin with my own data a javascript error comes up:
![image](https://user-images.githubusercontent.com/38167453/87136227-7c56bc80-c29b-11ea-9659-4cc11447a2cc.png)

Do you have any ideas?

Many Thanks,

Birthe

**Comment by RonnyWeiss** [comment]
Hello

Yes maybe it's a bug. Culd you please send me more information or build me an example on apex.oracle.com where you can reproducde the error? what is you sql statement? 

Kind regards

Ronny

**Comment by BirtheG** [comment]
Hello Ronny,

I have created a workspace on apex.oracle.com.

Workspace: Birthe
Username: orcale
pwd: APEX1234!

Go into your application. I have added a custom chart on page 13.

Thanks.

Birthe

**Comment by RonnyWeiss** [comment]
Hello

Your Problem was wrong Name ofid column it was names elemnt_id by you. So now it’s working.

Kind regards

Ronny

Gesendet von Mail<https://go.microsoft.com/fwlink/?LinkId=550986> für Windows 10

Von: Birthe<mailto:notifications@github.com>
Gesendet: Sonntag, 12. Juli 2020 15:56
An: RonnyWeiss/Apex-D3-Organization-Chart<mailto:Apex-D3-Organization-Chart@noreply.github.com>
Cc: Ronny Weiss<mailto:ronnyweiss@outlook.com>; Comment<mailto:comment@noreply.github.com>
Betreff: Re: [RonnyWeiss/Apex-D3-Organization-Chart] Get Javascript error during rendering (#4)


Hello Ronny,

I have created a workspace on apex.oracle.com.

Workspace: Birthe
Username: orcale
pwd: APEX1234!

Go into your application. I have added a custom chart on page 13.

Thanks.

Birthe
